src/components/sandbox/AngularReactiveSerializer.ts

```diff
@@ -1,7 +1,7 @@
 import { ComponentBinding } from "./ComponentBinding";
 import { BaseSerializer, Serializer, SerializerState } from "./BaseSerializer";
 
-const ReactiveComponents = [
+export const ReactiveComponents = [
```

Comment:
should no longer need this

Reply:
added modifyProps to the interface just like postProcess, therefore no longer need this export.